src/main/java/com/git/dummies/cc/Application.java

```diff
@@ -12,6 +12,12 @@ public static void main (String[] args) {
         words.add("dummies");
         words.add("now");
 
+        afiseaza();
+
         System.out.println(words);
     }
+
+    public static void afiseaza() {
+        System.out.println("pentru un nou commit");
```

Comment:
vreau mesaj in engleza

Reply:
Christophe. c'est pas pour toi. je fais un cours (GIT) de junieurs dans l'entreprise ( H Grup) et je voulais juste montrer comment github marche et les pulls requests.
Par contre, ta traduction retraduit = Je ne vois pas ou c'est le test de l'unite 😄

Reply:
j'espere que tu n'utilise pas mon tres secure et business code 😏